skbio/sequence/_dna.py

```diff
@@ -132,6 +142,35 @@ def degenerate_map(cls):
             "H": set("ACT"), "V": set("ACG"), "N": set("ACGT")
         }
 
+    @classproperty
+    def default_gap_char(cls):
+        """Gap character to use when constructing a new gapped sequence.
```

Comment:
The docstring will be inherited if you use `overrides`:

```python
@classproperty
@overrides(GrammaredSequence)
def default_gap_char(cls):
    return '-'
```

Same comment below for `gap_chars`.